### PR TITLE
Improve Slack notification message and fix leakages

### LINKED ISSUE #742: Capture and send Alert data to incoming webhooks

### PR DESCRIPTION
**Pull request by DementevNikita**
**What changes did you make?** (Give an overview)
This PR contain improvements for Slack/Webhook notifications. Several bugs and leakages were fixed. Slack message formatting has been improved.

**How to test**
The best way would be:
1. Create Slack workspace and create an incoming webhook.
2. Configure the platform to send notifications via Slack
3. Inject complex data graph and emulate several Alert types using Ingestion API (remove an existing column, inject failed job run/test run)
4. Assert that Slack messages appear and they are providing correct data

closes #742

### CODE REVIEW

File: odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/notification/processor/message/EmojiUtils.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package org.opendatadiscovery.oddplatform.notification.processor.message;
+
+public class EmojiUtils {
```
Comment (damirabdul) [inline odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/notification/processor/message/EmojiUtils.java:3]: you can use @UtilityClass here

File: odd-platform-api/src/main/resources/application.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 spring:
   datasource:
-    url: jdbc:postgresql://127.0.0.1:5432/odd-platform
+    url: jdbc:postgresql://127.0.0.1:6432/odd-platform
```
Comment (damirabdul) [inline odd-platform-api/src/main/resources/application.yml:3]: please return 5432

File: odd-platform-api/src/main/resources/application.yml [hunk 3]
```diff
@@ -96,4 +103,4 @@ logging:
     org.jooq.tools.LoggerListener: info
     io.r2dbc.postgresql.QUERY: info
     io.r2dbc.postgresql.PARAM: info
-    org.opendatadiscovery.oddplatform.notification: info
\ No newline at end of file
+    org.opendatadiscovery.oddplatform.notification: debug
```
Comment (damirabdul) [inline odd-platform-api/src/main/resources/application.yml:106]: please revert

**Review by damirabdul — COMMENTED** [review]
LGTM

**Review by germanosin — APPROVED** [review]
LGTM

**Review by damirabdul — APPROVED** [review]

**Pull request merged**